### ISSUE TITLE
Learning rate of ImageNet Training

### ISSUE THREAD

**Issue by LeeDoYup** [google-research/fixmatch#55]
I found that the lr is decayed at every 50 epoch.
However, it is different to the description in the paper (60, 120, 160, 200 epochs)

**Comment by AlexeyKurakin** [comment]
The results reported in the paper were actually using (60, 120, 160, 200 epochs) learning rate decay schedule.
After paper was initially submitted to the conference, we kept tweaking some of hyperparameters and included slightly updated version of hyperparameters in the code.
So if will you use hyperparameters from the paper you should get the same results as in the paper.
If you will use hyperparameters from the code you should get either similar or slightly (fraction of percent) better results.
